TASVideos.Core/Services/QueueService.cs

```diff
@@ -400,6 +400,7 @@ public async Task<string> MapParsedResult(IParseResult parseResult, Submission s
 		submission.System = system;
 		submission.CycleCount = parseResult.CycleCount;
 		var warnings = parseResult.Warnings.ToList();
+		submission.Warnings = null;
```

Comment:
what's going on here?

Reply:
Warnings string was created upon submission creation, and then persisted on updates. I'm clearing it now so the newly parsed movie is the only one involved in warning creation.